README.adoc

```diff
@@ -1086,6 +1086,21 @@ Customer.where("purchases_count > :min AND purchases_count <= :max", min: 0, max
 
 NOTE: Rails 6.0 or later is required for endless range Ruby 2.6 syntax, and Rails 6.0.3 for beginless range Ruby 2.7 syntax.
 
+=== `where.not` with multiple attributes
+
+Avoid passing multiple attributes to `where.not`. Rails logic in this case has changed in Rails 6.1 and
+will now yield results matching either of those conditions,
+e.g. `where.not(status: 'active', plan: 'basic')` would return records with active status when the plan is business.
+
+[source, ruby]
+----
+# bad
+User.where.not(status: "active", plan: "basic")
+
+# good
+User.where.not("status = ? AND plan = ?", "active", "basic")
```

Comment:
`NOT (... AND ...)` is a direct replacement for `where.not` with multiple conditions, i.e. it would return records with active statu on the business plan.

The cop [suggests](https://github.com/rubocop/rubocop-rails/commit/7d03a57db63e3a89e723082705561f75304a4d21#diff-9074cb50add7c36de7890a2002f6f1d11b5bb582594ebc75b714524b8ebd33e7R23-R24) multiple chained statements and a `NOT (... OR ...)` to match the older Rails behaviour.
But suggesting this would need a clarification.